todo_app/.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```

Comment:
all the `/.idea/` files should be added to `.gitignore`

Reply:
added